LambdaRuntimeDockerfiles/Infrastructure/src/Infrastructure/SelfMutatingPipelineStack.cs

```diff
@@ -65,9 +65,19 @@ internal SelfMutatingPipelineStack(
                 { "AWS_LAMBDA_DOTNET_FRAMEWORK_CHANNEL",
                     new BuildEnvironmentVariable { Type = BuildEnvironmentVariableType.PLAINTEXT, Value =
                     System.Environment.GetEnvironmentVariable("AWS_LAMBDA_DOTNET_FRAMEWORK_CHANNEL") ?? string.Empty } },
+                { "AWS_LAMBDA_GITHUB_REPO_OWNER_STAGING",

```

Comment:
i didnt bother to move all of these to a json file yet because we would still need to send in the token key as env variables anyway. i may come back to this in the future